### ISSUE TITLE
addWebpackModuleRule doesn't work

### ISSUE THREAD

**Issue by hellomrbigshot** [arackaf/customize-cra#153]
I want to add 'raw-loader' or 'text-loader' to import markdown files.
I tried
```
const { override, addWebpackModuleRule } = require('customize-cra');
module.exports = override(
  addWebpackModuleRule(
    {
      test: /\.md$/,
      use: { loader: 'raw-loader' }
    }
  )
)
```
It doesn't work. It only shows the path instead of the content of the md files.

I also tried
```
module.exports = override(
  addWebpackModuleRule(
    {
      test: /\.md$/,
      use: { loader: require.resolve('raw-loader') }
    }
  )
)
```

or

```
const { override, addWebpackModuleRule } = require('customize-cra');
module.exports = override(
  config => ({
    ...config,
    output: {
      ...config.output,
      globalObject: 'this'
    },
  }),
  addWebpackModuleRule(
    {
      test: /\.md$/,
      use: { loader: 'row-loader' }
    }
  )
)

```
But it still doesn't work.

Then I run 
```
yarn eject
```
and add ```raw-loader``` into webpack.config.js
```
{
     test: /\.md$/,
     loader: require.resolve('text-loader')
}
```
it works.

And should api.md from
```
module.exports = {
  override(
    addWebpackModuleRule({text: /\.txt$/, use: 'raw-loader'})
  )
}
```
change to
```
// text -> test
module.exports = override(
    addWebpackModuleRule({test: /\.txt$/, use: 'raw-loader'})
)
```

**Comment by with-heart** [comment]
Are you saying your last example (changing text -> test) is working for you?

**Comment by with-heart** [comment]
The `api.md` example is fixed in d1bd182.

**Comment by hellomrbigshot** [comment]
> Are you saying your last example (changing text -> test) is working for you?

No, it still doesn't work 😂😂😂

**Comment by postgetme** [comment]
same issue

**Comment by with-heart** [comment]
Published a fix for this under `v0.6.1` thanks to @postgetme's work in #164.
